CONTRIBUTING.md

```diff
@@ -114,6 +114,13 @@ pip install -e $SRF_HOME/build/python
 pytest $SRF_HOME/python
 ```
 
+### Building API Documentation
+From the root of the SRF repo, configure CMake with `SRF_BUILD_DOCS=ON` then build the `srf_docs` target. Once built the documentation will be located in the `build/docs/html` directory.
+```bash
+cmake -B build -DSRF_BUILD_DOCS=ON .
+cmake --build build --target srf_docs
+```
+
```

Comment:
This seems unrelated. Is this intentional?

Reply:
That is weird! That is the commit from #183 not sure how it got in this pr.

Reply:
Looks like it just got branched from the wrong commit